packages/cli/src/platform/namespace.ts

```diff
@@ -0,0 +1,19 @@
+// import { Command as MoltCommand } from '@molt/command'
+import { Command } from '@prisma/internals'
+
+export class Platform implements Command {
+  public static new(): Platform {
+    return new Platform()
+  }
+
+  public async parse(argv: string[]) {
+    // todo Molt Command does not support CJS
+    // const args = MoltCommand.create().parse({ line: argv })
```

Comment:
I would like to use `@molt/command` for parameter definition, validation, and help rendering.

It currently relies on an ESM only package though, so I need to go "fix" that, because IIUC @Jolg42 we're not going to be using ESM for the foreseeable future still for the CLI right?

Reply:
@millsp is actually working on something about that right now, watch out for https://github.com/prisma/prisma/pull/22017 I hope we can merge this soon.

Reply:
From a step back, as a general advice, I think it's important, from a user point of view, that the CLI doesn't completely look and feel different for platform commands.

Reply:
I need to get this comment resolved to merge. I think this thread is informational now and nothing actionable. Will resolve!